### ISSUE TITLE
ReferenceError with project name containing hyphen/dash

### ISSUE THREAD

**Issue by dawranliou** [shadow-cljs/lein-template#2]
I'd like to report this issue although I'm not sure if this is an issue with the shadow-cljs itself or the lein template.
I created a rum project using this command
```
lein new shadow-cljs rum-workshop +rum
````
and when I launched the app with command

```
shadow-cljs watch app
```
the app did not mounted on the div. In the console I saw the error: `ReferenceError: workshop is not defined`

Then I created another project with `lein new shadow-cljs rumworkshop +rum` and it worked.

The versions on my machine are:

```
cli version: 2.4.26  node: v10.7.0
```

Thanks.

**Comment by thheller** [comment]
The problem most likely is in the generated `index.html` file.

You'll find a `<script>rum-workshop.<...></script>` which is not correctly munged. You can change it to `<script>rum_workshop...</script>` manually to fix it for now.

**Comment by dawranliou** [comment]
Oh cool. Yes that solves the problem for me. Thanks for your quick response!

**Comment by thheller** [comment]
Fixed in `0.1.3`.
